.circleci/config.yml

```diff
@@ -1162,6 +1162,14 @@ jobs:
           command: |
             nvm install
             nvm use && node --version && npm --version
+      - run:
+          name: Install Rust
+          command: |
+            curl https://sh.rustup.rs -sSf | sh -s -- -y
+      - run:
+          name: Install Just
+          command: |
+            cargo install just
```

Comment:
Will building just in CI every time slow things down a lot? We could alternatively install a pre-built binary: https://just.systems/man/en/chapter_5.html